cfme/tests/containers/test_ad_hoc_metrics.py

```diff
@@ -16,6 +17,13 @@
                          scope='function')
 ]
 
+TEST_ITEMS = [ContainersProviderCollection, NodeCollection]
```

Comment:
I am thinking to change this to '''TEST_ITEMS = ['containers_providers', 'container_nodes']```

Reply:
I need the collection objects, is ```container_nodes``` and ```containers_providers``` is a collection object?

Reply:
Its not a collection object, but combined with @jawatts comment below, its the name of the collection that you want to get from the appliance.

Reply:
@mshriver amm interesting, you suggest to have the name of the required collection, and use ```getattr(app.collections, <collection name>)``` to actually get it?

Reply:
This is how the function ```get_test_object``` function looks like:

```python
TEST_ITEMS = ["containers_providers", "container_nodes"]


def get_test_object(appliance, provider, test_item):
    test_item = getattr(appliance.collections, test_item)
    return (
        provider if test_item is ContainersProviderCollection else test_item(appliance).all().pop())
```

Reply:
@shalomnaim1  close - but  the call to instantiate test_item and pass it appliance `test_item(appliance)` doesn't look right at all, as appliance.collections.test_item will return you a collection instance, not a collection class.